src/transformers/processing_utils.py

```diff
@@ -874,7 +874,11 @@ class MyProcessingKwargs(ProcessingKwargs, CommonKwargs, TextKwargs, ImagesKwarg
         else:
             # kwargs is a flat dictionary
             for key in kwargs:
-                if key not in used_keys:
+                if key not in ModelProcessorKwargs.__annotations__["common_kwargs"].__annotations__.keys():
```

Comment:
I see! Good catch, no simpler way of catching these keys? (I assume not because of python version dependencies)

Reply:
I don't thinks so, I used the same format as the tests for catching modality keys just above, which seems to be robust across python versions